roles/openshift_node/tasks/systemd_units.yml

```diff
@@ -9,6 +9,7 @@
   template:
     dest: "/usr/lib/tmpfiles.d/cleanup-cni.conf"
```

Comment:
```suggestion
    dest: "/etc/tmpfiles.d/cleanup-cni.conf"
```

This should be both RHEL and Atomic compatible

Reply:
Agree. But I do not have the big picture, if changing the path is valid.